agent/testagent.go

```diff
@@ -353,6 +353,8 @@ func TestConfig(sources ...config.Source) *config.RuntimeConfig {
 				enabled = true
 				ca_config {
 					cluster_id = "` + connect.TestClusterID + `"
+                	private_key_type = "ec"
```

Comment:
nit: is there a whitespace issue here or just GitHub having a bad day?

Reply:
It passed `go fmt` so I'm assuming it's Github being weird.